spring-ai-model/src/main/java/org/springframework/ai/util/json/schema/JsonSchemaGenerator.java

```diff
@@ -140,6 +140,8 @@ public static String generateForMethodInput(Method method, SchemaOption... schem
 				required.add(parameterName);
 			}
 			ObjectNode parameterNode = SUBTYPE_SCHEMA_GENERATOR.generateSchema(parameterType);
+			// Remove OpenAPI format as some LLMs (like Mistral) don't handle them.
+			parameterNode.remove("format");
```

Comment:
It would be good to find out what other cases the OpenAPI format can be used in the context of ChatClient request.